include/psa/crypto_values.h

```diff
@@ -39,7 +39,32 @@
  * @{
  */
 
-/* PSA error codes */
+/* Status codes are shared across PSA modules. All PSA modules
+ * that define a given status code must give it the same numerical value
+ * and must use the same sequence of tokens and (lack of) whitespace between
+ * tokens. This is necessary to allow the same C source file to include
+ *  multiple PSA specifications' headers.
```

Comment:
```suggestion
 * multiple PSA specifications' headers.
```

Reply:
Fixed by amending the commit, since I rebased it anyway.